src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -271,22 +261,34 @@ public void setName(String n)
      *
      * @return      Chunk Coordinates of the center of the colony
      */
-    public BlockPos getCenter() { return center; }
+    public BlockPos getCenter()
+    {
+        return center;
+    }
 
     /**
      * Marks the instance dirty
      */
-    private void markDirty() { isDirty = true; }
+    private void markDirty()
+    {
+        isDirty = true;
+    }
 
     /**
      * Marks citizen data dirty
      */
-    public void markCitizensDirty() { isCitizensDirty = true; }
+    public void markCitizensDirty()
+    {
+        isCitizensDirty = true;
+    }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)